src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -22,6 +22,19 @@
 
 namespace Tizen.NUI.Components
 {
+    /// <summary>
+    /// PoppedEventArgs is a class to record popped event arguments which will be sent to user.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class PoppedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Page popped by Navigator.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Page page;
```

Comment:
```suggestion
        public Page Page {get;set}
```

Reply:
Thank you for your review!
I have been updated the code.